schema.sql

```diff
@@ -1,47 +1,30 @@
 /* Database schema to keep the structure of entire database. */
 
- create table owners (
+ create table vets(
  id int generated always as identity,
- full_name varchar(100),
+ name varchar (100),
  age int,
- primary key (id)
-);
+ date_of_graduation date,
+ primary key (id));
+
+
+ create table specializations(
+ species_id int,
+ vets_id int,
+ primary key (species_id,vets_id));
+
+ALTER TABLE public.specializations ADD CONSTRAINT specializations_fk FOREIGN KEY (species_id) REFERENCES public.species(id);
+ALTER TABLE public.specializations ADD CONSTRAINT specializations_fk_1 FOREIGN KEY (vets_id) REFERENCES public.vets(id);
+
+
+ create table visits (
+ animals_id int,
+ vets_id int,
+ date_of_visits date,
+ primary key (animals_id,vets_id,date_of_visits));
+
+
+ALTER TABLE public.visits ADD CONSTRAINT visits_fk FOREIGN KEY (animals_id) REFERENCES public.animals(id);
+ALTER TABLE public.visits ADD CONSTRAINT visits_fk_1 FOREIGN KEY (vets_id) REFERENCES public.vets(id);
+
```

Comment:
- [ ] [OPTIONAL] I would advise sticking to a particular letter casing for keywords. So instead of using lowercase and uppercase in different places, I would advise using uppercase all through. This also improves readability.